### PR TITLE
Refactor to prefer baseZ instead of height for big Z

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Small refactor that adjust parameter names.

### CODE REVIEW

File: src/openrct2/actions/LargeSceneryPlaceAction.hpp [hunk 1]
```diff
@@ -364,21 +365,21 @@ DEFINE_GAME_ACTION(LargeSceneryPlaceAction, GAME_COMMAND_PLACE_LARGE_SCENERY, La
...
-            int32_t height = surfaceElement->GetBaseZ();
+            int32_t baseZ = surfaceElement->GetBaseZ();
             int32_t slope = surfaceElement->GetSlope();
 
-            if (slope & 0xF)
+            if ((slope & TILE_ELEMENT_SLOPE_ALL_CORNERS_UP) != TILE_ELEMENT_SLOPE_FLAT)
             {
-                height += 16;
-                if (slope & 0x10)
+                baseZ += 16;
```
Comment (Gymnasiast) [inline src/openrct2/actions/LargeSceneryPlaceAction.hpp]: LAND_HEIGHT_STEP

File: src/openrct2/paint/tile_element/Paint.Path.cpp [hunk 2]
```diff
@@ -911,28 +911,28 @@ void path_paint(paint_session* session, uint16_t height, const TileElement* tile
...
 
-            sub_98196C(session, imageId | patrolColour << 19 | IMAGE_TYPE_REMAP, 16, 16, 1, 1, 0, height2 + 2);
+            sub_98196C(session, imageId | patrolColour << 19 | IMAGE_TYPE_REMAP, 16, 16, 1, 1, 0, patrolAreaBaseZ + 2);
         }
     }
 
     if (session->ViewFlags & VIEWPORT_FLAG_PATH_HEIGHTS)
     {
-        uint16_t height2 = 3 + tile_element->GetBaseZ();
+        uint16_t heightMarkerBaseZ = 3 + tile_element->GetBaseZ();
```
Comment (Gymnasiast) [inline src/openrct2/paint/tile_element/Paint.Path.cpp]: Nitpick, but
```suggestion
        uint16_t heightMarkerBaseZ = tile_element->GetBaseZ() + 3;
```
is easier to follow IMO.

File: src/openrct2/actions/LargeSceneryPlaceAction.hpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/openrct2/actions/LargeSceneryPlaceAction.hpp]: One day I'll go round and refactor all the million instances of this code to call a function.

**Comment by Gymnasiast** [comment]
You have added a td6 file. Was that intentional?

**Comment by duncanspumpkin** [comment]
> You have added a td6 file. Was that intentional?

No. (This is the last time I use git commands I don't fully understand)

**Pull request merged**